doc/meetings/2017-05-15.md

```diff
@@ -0,0 +1,69 @@
+# Node.js LTS meeting 05 May 2017
+
+- Github issue: https://github.com/nodejs/LTS/issues/213
+- Meeting Video: https://youtu.be/UFBMF9ndBDo
+- Next meeting: 05 June 2017
+- Previous meetings: https://github.com/nodejs/LTS/issues/201
+
+## Present
+
+- Michael Dawson (@mhdawson)
+- Myles Borins (@MylesBorins)
+- Jeremiah Senkpiel (@fishrock123)
+- Sam Roberts (@sam-github)
+- Gibson Fahnestock (@gibfahn)
+
+
+## Agenda
+
+### nodejs/LTS
+- meta: updated messaging regarding dates [#141](https://github.com/nodejs/LTS/pull/141)
+- Clarify what happens with odd-numbered releases in April [#128](https://github.com/nodejs/LTS/pull/128)
+- meta: charter the LTS Working Group [nodejs/CTC#122](https://github.com/nodejs/CTC/pull/122)
+- Potential Semver Minor Backports [#177](https://github.com/nodejs/LTS/issues/188)
+
+
+## Minutes
+
+### meta: updated messaging regarding dates [#141](https://github.com/nodejs/LTS/pull/141)
+
+All: No objections
+
+
+### Clarify what happens with odd-numbered releases in April [#128](https://github.com/nodejs/LTS/pull/128)
+
+- Myles: I’m not sure we need to declare a support statement at all, we
+  definitely don’t want to call it maintenance, that conflates it with LTS
+  maintenance mode, which is different.
+- Sam: if we describe odd-numbered as stable then do we need maintenance?
+- Myles: we shouldn’t call odd-numbered releases stable, they are current
+  because they don’t have a support process.
+- Myles: We could say: “After the next Current release line comes out, there
+  will be no more scheduled releases. Further releases will be agreed on a
+  case-by-case basis.”
+- All: agreed
+- Myles: I think the bigger issue is that we are inconsistent about how stable
+  current is, and whether it’s recommended for more general use.
+- Myles to raise CTC issue
```

Comment:
Are you talking about this line (`Myles to raise CTC issue`) or 

>Myles: We could say: “After the next Current release line comes out, there will be no more scheduled releases. Further releases will be agreed on a case-by-case basis.”

?

The CTC issue is a more general question about whether we recommend people use Current releases. 

If you want to change what it says that Myles said on L45, then feel free to just edit this PR!

![image](https://cloud.githubusercontent.com/assets/15943089/26165964/c402cc2e-3b29-11e7-9736-8dc9137a5c33.png)